openfisca_core/parameters.py

```diff
@@ -741,3 +761,7 @@ def contains_nan(vector):
         return any([contains_nan(vector[name]) for name in vector.dtype.names])
     else:
         return np.isnan(np.min(vector))
+
+
+def indent(text):
```

Comment:
Is it the right place to have this function ? It is used in other places and called through `from .parameters import indent  # Need to import here to avoid a circular dependency`in taxscales.py

Reply:
Yep, very right, doing this ugly stuff usually means that the function is defined in the wrong place.